sherpa_contrib/marx.py

```diff
@@ -203,9 +204,9 @@ def plot_marx_spectrum(id=None, elow=None, ehigh=None, ewidth=None,
     hplot.title = 'MARX Spectrum: {}'.format(vals['model'])
 
     # LaTeX support depends on the backend
-    if backend.name == 'pylab':
+    if plot.backend.name == 'pylab':
         hplot.ylabel = 'Flux (photon cm$^{-2}$ s$^{-1}$ keV$^{-1}$)'
-    elif backend.name == 'chips':
+    elif plot.backend.name == 'chips':
```

Comment:
I think we can remove this! GH annoyingly won't allow me to suggest the removal as line 209 wasn't changed in this PR...